src/chttpd/src/chttpd_db.erl

```diff
@@ -1432,6 +1433,7 @@ couch_doc_open(Db, DocId, Rev, Options0) ->
 db_attachment_req(#httpd{method='GET',mochi_req=MochiReq}=Req, Db, DocId, FileNameParts) ->
     FileName = list_to_binary(mochiweb_util:join(lists:map(fun binary_to_list/1,
         FileNameParts),"/")),
+    validate_attachment_name(FileName),
```

Comment:
why validate on GET?

Reply:
Just to be consistent in the API response and return 400 when the attachment name is not valid. 
We can remove the validation and just return 404

Reply:
@rnewson Do you think that we should return 404 and not 400 when a not allowed attachment name is requested?